gh-actions/common/build-debian/action.yml

```
@@ -79,6 +93,11 @@ runs:
           if [ -n "${GITHUB_TOKEN}" ]; then
             git config --system url."https://api:${GITHUB_TOKEN}@github.com/".insteadOf "https://github.com/"
           fi
+    - name: Run lintian
+      if: inputs.lintian != 'false'
+      shell: bash
+      working-directory: ${{ env.SOURCE_OUTPUT_DIR }}
+      run: lintian ${{ inputs.lintian }} -- *.changes
```

```suggestion
      run: lintian --pedantic --fail-on error ${{ inputs.lintian }} -- *.changes
```

So we still see what's not exactly _clean_. Plus, IMHO, it would be nice to have an option to control `--fail-on`, since ideally good packages should set it to `warning`

---

Oh, I sorry ignore me on the `--fail-on` thingy... It's indeed possible to override the value.

Although, maybe this option is even too powerful, since IMHO any suppression should be in sources.